.github/workflows/tests.yml

```diff
@@ -127,7 +127,29 @@ jobs:
           git log --oneline -1 ${{ env.TARGET_BRANCH_NAME }}
           git rebase ${{ env.TARGET_BRANCH_NAME }}
 
+      - name: Check if container rebuild is needed
+        id: need_rebuild
+        run: |
+          changed_files=$(git diff --name-only ${{ env.TARGET_BRANCH_NAME }}..HEAD)
+          echo -e "Changed files: \n$changed_files\n"
+
+          . .structure-config
```

Comment:
This could be a potential attack source.

In your PR you can just change the `.structure-config` file and you can do anything to the workflow here.

However, it's probably not that problematic because we have to confirm tests for 3rd party contributors. Just want to point this out.

Reply:
Ah, right - we do the same but on `pull_request_target` so it's really different https://github.com/rhinstaller/anaconda/blob/master/.github/workflows/infra-check.yml#L63 Thanks for the note!